moto/elbv2/models.py

```diff
@@ -283,6 +285,8 @@ def update_from_cloudformation_json(
         certificates = properties.get("Certificates")
 
         default_actions = elbv2_backend.convert_and_validate_properties(properties)
+        if certificates:
+            certificates = elbv2_backend.convert_and_validate_certificates(certificates)
```

Comment:
Is it normal in this codebase to duplicate conditions like this? Or should this be DRY'd out. [See here](https://github.com/sahilshah6196/moto/pull/3/files#diff-9dd6ca137d3431e1a7f99285e62db5459bc57d8827dc13189c28f858cd7224ccR268-R269) for the duplication.

Reply:
I could move it into that function

Reply:
8dee7c3